static/assets/geojson/hsl_zone_areas_20190508.geojson

```diff
@@ -1,3789 +0,0 @@
-{
```

Comment:
We should stay compatible to hol and not delete their geojson... (same for the files below)